src/config/templates.toml

```
@@ -45,11 +75,8 @@ label(if(current_operation, "current_operation"),
   ),
 )
 '''
+builtin_op_log_comfortable = 'builtin_op_log_compact + "\n"'
```

```suggestion
builtin_op_log_comfortable = 'builtin_op_log_compact ++ "\n"'
```

Otherwise this cannot be expanded.